beacon-chain/blockchain/process_block.go

```diff
@@ -75,6 +75,23 @@ func (s *Service) postBlockProcess(cfg *postBlockProcessConfig) error {
 	defer reportProcessingTime(startTime)
 	defer reportAttestationInclusion(cfg.roblock.Block())
 
+	// Check for equivocation before inserting into fork choice
+	slashing, slashingErr := s.detectEquivocatingBlock(cfg.ctx, cfg.roblock)
+	if slashingErr != nil {
+		return errors.Wrap(slashingErr, "could not detect equivocating block")
```

Comment:
this should probably log instead of returning and preventing FCU? `could not check for block equivocation` seems more accurate